juristische-zitierweise.csl

```diff
@@ -60,7 +60,7 @@
         <names variable="editor">
           <name form="short" delimiter="/" initialize-with=""/>
         </names>
-        <text variable="title"/>
+        <text variable="container-title" form="short"/>
```

Comment:
I added manually a `container-title-short` but this did not show up here. Is that forbidden for a chapter?

Reply:
No, CSL 1.0.1 puts no restrictions on which fields can be used with which item types. Might be some kind of bug in the editor. (if it's not too much work, it might help if you could give the exact steps to reproduce this problem)

Reply:
Okay, I created an item of type chapter in Zotero and export it to CSL JSON, added there the field `container-title-short` with a value and then import that (without the outermost parenthesis for a set) into the CSL editor, where I don't see any effect of the short container title.

I used this CSL-JSON data
```json
{
	"id": "http://zotero.org/users/96641/items/TFNFUVLI",
	"type": "chapter",
	"title": "Endnotes",
	"container-title": "The World is Open",
	"container-title-short": "TWO",
	"publisher": "Wiley-Blackwell",
	"page": "427-467",
	"source": "Wiley Online Library",
	"URL": "https://onlinelibrary.wiley.com/doi/abs/10.1002/9781118269381.notes",
	"ISBN": "978-1-118-26938-1",
	"note": "DOI: 10.1002/9781118269381.notes",
	"language": "en",
	"issued": {
		"date-parts": [
			[
				"2012"
			]
		]
	},
	"accessed": {
		"date-parts": [
			[
				"2018",
				7,
				2
			]
		]
	}
}
```

Any idea?

Reply:
This seems to work fine for me. I loaded this JSON into the visual editor, and edited the style to use

```xml
<text variable="container-title-short"/>
```

which prints "TWO" for the bibliography:

<hr/>

<img width="408" alt="screen shot 2018-08-11 at 01 04 20" src="https://user-images.githubusercontent.com/77951/43988374-110c7ef4-9d24-11e8-91c4-43e436a44488.png">

Reply:
Okay, and what happens if you take instead
```
<text variable="container-title" form="short"/>
```
?

Reply:
<img width="467" alt="screen shot 2018-08-11 at 12 52 13" src="https://user-images.githubusercontent.com/77951/43994114-f28c3dbc-9d86-11e8-9efc-166e14958da7.png">

This looks like a citeproc-js mapping issue. `<text variable="container-title" form="short"/>` pulls in the (journal title) abbreviation for the first item, which is stored as "journalAbbreviation" in the CSL JSON (https://github.com/citation-style-language/csl-editor/blob/1319eeaebb5ca13f001d7773f8cc30a923718466/src/exampleData.js#L56). But "container-title-short" seems to be ignored.

This seems like a similar situation to what we had with "shortTitle" and "title-short". I can raise the issue with Frank.

Reply:
> This seems like a similar situation to what we had with "shortTitle" and "title-short". I can raise the issue with Frank.

That would be great. Thank you!

Anything we should do in this PR or can it be merged now?

Reply:
> Anything we should do in this PR or can it be merged now?

Well, `<text variable="container-title" form="short"/>` and `<text variable="container-title-short"/>` are both correct CSL in this case. And as Zotero currently doesn't offer a variable for chapters (https://aurimasv.github.io/z2csl/typeMap.xml#map-bookSection), it doesn't really matter which one we pick here.

Reply:
Do both variants use `container-title` if the short form is not present?

Do you suggest to switch the form here? Maybe I am missing something else here...?

Reply:
> Do both variants use container-title if the short form is not present?

I think only the `form="short"` call has fallback to the "long" form if unavailable. See http://docs.citationstyles.org/en/1.0.1/specification.html#text. Pretty sure the CSL specification is silent on fallback for "container-title-short".

Reply:
Okay. How do you want to finish this PR then? (I would like to complete this soon as I have promised to some PhD candidates here...)